src/daft-sql/src/planner.rs

```diff
@@ -670,7 +680,34 @@ impl SQLPlanner {
             SQLExpr::GroupingSets(_) => unsupported_sql_err!("GROUPING SETS"),
             SQLExpr::Cube(_) => unsupported_sql_err!("CUBE"),
             SQLExpr::Rollup(_) => unsupported_sql_err!("ROLLUP"),
-            SQLExpr::Tuple(_) => unsupported_sql_err!("TUPLE"),
+            // Similar to rust and python conventions, tuples always have a fixed size,
+            // so we convert them to a fixed size list.
+            SQLExpr::Tuple(exprs) => {
```

Comment:
How are tuples in SQL used? As in, do we have a supported SQL query/expression using tuples right now? Just curious

Reply:
yes, this here updates our sql support to include support for tuples. 

you can use them anywhere exprs are accepted now.

```py
fsl_from_sql = daft.sql_expr('(1, 2, 3)')
```
this is the functional equivalent of 
```py
fsl_from_python = daft.lit(
    (1, 2, 3)
).cast(daft.DataType.fixed_size_list(daft.DataType.int64(), 3))
```